api/v1beta1/packetcluster_webhook.go

```diff
@@ -47,6 +47,23 @@ func (c *PacketCluster) Default() {
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (c *PacketCluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", c.Name)
+	allErrs := field.ErrorList{}
+
+	// Must have either one of Metro or Facility
+	if len(c.Spec.Facility) == 0 && len(c.Spec.Metro) == 0 {
+		allErrs = append(allErrs,
+			field.Invalid(field.NewPath("spec", "Metro"),
+				c.Spec.Metro, "field is required when Facility is not set"),
```

Comment:
I think it would be preferable to say `field is required` for `metro` if neither option is set; that way we guide the user towards picking the right option to set.